modules/unsupported/geojsonstore/pom.xml

```diff
@@ -28,14 +24,29 @@
 		<dependency>
 			<groupId>commons-io</groupId>
 			<artifactId>commons-io</artifactId>
-			<version>2.6</version>
 		</dependency>
 		<dependency>
 			<groupId>org.geotools</groupId>
 			<artifactId>gt-sample-data</artifactId>
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-databind</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.bedatadriven</groupId>
```

Comment:
Hi @ianturton You may want to look into using the forked version from the graphhopper team that has been upgraded wr/ JTS 

eg. 
```xml
<dependency>
  <groupId>com.graphhopper.external</groupId>
  <artifactId>jackson-datatype-jts</artifactId>
  <version>[latest]</version>
</dependency> 
```

see: https://github.com/graphhopper/jackson-datatype-jts

bedatadriven/jackson-datatype-jts seems to have been [abandoned](https://github.com/bedatadriven/jackson-datatype-jts/pull/18#issuecomment-470858490)